### ISSUE TITLE
Several whitespace and newline rules broken for less (9.7.1)

### ISSUE THREAD

**Issue by FreekVR** [stylelint/stylelint#3780]
Since the 9.7.0 release, several white-space related rules are giving false positives.

> Which rule, if any, is the bug related to?

- at-rule-semiolon-newline-after
- declaration-colon-space-after
- block-closing-brace-newline-after

... Probably many others

> What CSS is needed to reproduce the bug?

`at-rule-semiolon-newline-after`
```less
// Custom styles
@import 'base/**/*.less';
@import 'layout/**/*.less';

@import '../app/patterns/atoms/**/*.less';
@import '../app/patterns/molecules/**/*.less';
@import '../app/patterns/organisms/**/*.less';
@import '../app/patterns/templates/**/*.less';
@import '../app/patterns/pages/**/*.less';
```

`declaration-colon-space-after`
```less
.someclass {
        &:extend(.link all, .link--primary all);
}
```

`block-closing-brace-newline-after`
```less
.font-size-h1 { .font-size(@font-size-mega, @font-size-mega, @font-size-mega, @font-size-giga); }    // 44px ... 44px - 72px
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "at-rule-semicolon-after": "always",
    "declaration-colon-space-after": "always",
    "block-closing-brace-newline-after": "always"
  }
}
```

> Which version of stylelint are you using?

9.7.0, 9.7.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Via CLI, but also occurs in Gulp

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Using less @import calls, but may affect vanilla css as well.

> What did you expect to happen?

Should pass rule.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
9:42  ✖  Expected newline after ";"   at-rule-semicolon-newline-after
(...) same for other rules
```

**Comment by jeddy3** [comment]
@FreekVR Thanks for the report and for using the template.

I think we've captured these regressions in https://github.com/stylelint/stylelint/issues/3754 and https://github.com/stylelint/stylelint/issues/3765. So I'll close this as a dupe.

If you have time, please consider helping out with these two issues as we rely on the contributions from external contributors to support Less in stylelint. I think both issues would benefit from being dug into so we can at least identify if the problem lies within stylelint or within the Less parser we use.
